### PR TITLE
SUBMARINE-1088. Page of How to Release

### PR DESCRIPTION
**Pull request by featherchen**
### What is this PR for?

Refer to https://inlong.apache.org/development/how-to-release
Create a new page describe how to release a new version of Submarine
### What type of PR is it?
Documentation 

### Todos
* [x] - Change inLong to Submarine

### What is the Jira issue?

https://issues.apache.org/jira/browse/SUBMARINE-1088
### How should this be tested?

Please help me double-check if there's something wrong.
### Screenshots (if appropriate)
Screenshot from 2021-11-20 15-48-56

### Questions:
* Do the license files need updating? No
* Are there breaking changes for older versions? No
* Does this need new documentation? No

### CODE REVIEW

File: website/docs/devDocs/HowToRelease.md [hunk 1]
```diff
@@ -0,0 +1,475 @@
...
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+
+> This article mainly introduces how the Release Manager releases a specific version of the project according to the Apache process.
+
+## 0. Preface
+
+Source Release is the focus of Apache’s attention and it is also a required content for release. Binary Release is optional, InLong can choose whether to release the binary package to the Apache warehouse or to the Maven central warehouse.
```
Comment (pingsutw) [inline website/docs/devDocs/HowToRelease.md:24]:
```suggestion
Source Release is the focus of Apache’s attention and it is also a required content for release. Binary Release is optional, Submarine can choose whether to release the binary package to the Apache warehouse or to the Maven central warehouse.
```
```suggestion
Source Release is the focus of Apache’s attention and it is also a required content for release. Binary Release is optional, InLong can choose whether to release the binary package to the Apache warehouse or to the Maven central warehouse.
```

File: website/docs/devDocs/HowToRelease.md [hunk 2]
```diff
@@ -0,0 +1,475 @@
...
+
+Please refer to the following link to find more ASF release guidelines:
+
+[Apache Release Guide](https://incubator.apache.org/guides/releasemanagement.html)
+
+[Apache incubator](https://incubator.apache.org/)
```
Comment (pingsutw) [inline website/docs/devDocs/HowToRelease.md:30]: We can remote it.

File: website/docs/devDocs/HowToRelease.md [hunk 3]
```diff
@@ -0,0 +1,475 @@
...
+
+The query results are as follows:
+
+### 1.5 Add your gpg public key to the KEYS file
+
+> SVN is required for this step
+
+The svn library of the DEV branch is https://dist.apache.org/repos/dist/dev/incubator/inlong
+
+The SVN library of the Release branch is https://dist.apache.org/repos/dist/release/incubator/inlong
```
Comment (pingsutw) [inline website/docs/devDocs/HowToRelease.md:146]:
```suggestion
The svn library of the DEV branch is https://dist.apache.org/repos/dist/dev/submarine

The SVN library of the Release branch is https://dist.apache.org/repos/dist/release/submarine
```
```suggestion
The svn library of the DEV branch is https://dist.apache.org/repos/dist/dev/incubator/inlong

The SVN library of the Release branch is https://dist.apache.org/repos/dist/release/incubator/inlong
```

**Review by pingsutw — COMMENTED** [review]
We should change the name from `inlong` to `submarine`
The links will be like https://dist.apache.org/repos/dist/release/submarine. 
we don't need to add "incubator" in the link.

**Comment by featherchen** [comment]
> Thanks for working on this We should change the name from `inlong` to `submarine` The links will be like https://dist.apache.org/repos/dist/release/submarine. we don't need to add "incubator" in the link.

@pingsutw  Thanks !
I have changed the context from inLong to Submarine, and also change the email template to the one we used in 0.6.0.

**Review by xunliu — APPROVED** [review]
@featherchen Thank you for your contribution.
If you are interested and time, would you be willing to participate in the release of the next version of submarine? You can see if you follow this document to see if it meets expectations. :-)

**Comment by featherchen** [comment]
> @featherchen Thank you for your contribution. LGTM If you are interested and time, would you be willing to participate in the release of the next version of submarine? You can see if you follow this document to see if it meets expectations. :-)

Sounds very interesting! It will be my pleasure if I can participate in the release of the next version of submarine.

**Review by pingsutw — APPROVED** [review]

**Pull request closed without merging**